pinot-broker/src/main/java/org/apache/pinot/broker/requesthandler/BaseSingleStageBrokerRequestHandler.java

```diff
@@ -140,7 +140,9 @@ public abstract class BaseSingleStageBrokerRequestHandler extends BaseBrokerRequ
   protected final int _defaultHllLog2m;
   protected final boolean _enableQueryLimitOverride;
   protected final boolean _enableDistinctCountBitmapOverride;
-  protected final int _queryResponseLimit;
+  protected final int _queryResponseLimitOverride;
```

Comment:
Let's keep the variable name unchanged. This is not only for override, but actually the cap of the allowed limit